src/condor/solvers/sweeping_gradient_method.py

```diff
@@ -38,12 +38,39 @@ def __init__(
         system,
         atol=1e-12,
         rtol=1e-6,
-        adaptive_max_step=0.0,
+        adaptive_min_steps=0,
+        rootfinder=brentq,
         max_step_size=0.0,
         nsteps=10_000.0,
+        **kwargs,
     ):
+        """
```

Comment:
I like having the options here instead of the implementations.sgm_trajectory since they are ~ a pass through. Having trouble setting up the docs stub.